projects/storefrontapp-e2e-cypress/cypress/helpers/accessibility/tabbing-order.config.ts

```diff
@@ -45,6 +45,22 @@ export const tabbingOrderConfig: TabbingOrderConfig = {
     { value: 'Submit', type: TabbingOrderTypes.BUTTON },
     { value: 'Cancel', type: TabbingOrderTypes.BUTTON },
   ],
+  cart: [
+    {
+      value: '/electronics-spa/en/USD/product/1981414/PL60%20Pink',
```

Comment:
I would go with `'/product/198....` only, instead of all of the prefixes

Reply:
Does not work because our helper expects an exact url